packages/ui/src/components/pages/Accounts/AccountControls/index.tsx

```diff
@@ -9,13 +9,19 @@ import { EventName, triggerEvent } from 'utils/eventemitter';
 enum AccountControlsAction {
   RenameAccount = 'Rename',
   RemoveAccount = 'Remove',
+  ShowAddressQrCode = 'Show QR Code',
```

Comment:
We should make it more clear here.
```suggestion
  ShowAddressQrCode = 'Show Address QR Code',
```